src/app/modules/main/module.nim

```diff
@@ -1671,12 +1671,6 @@ method addressWasShown*[T](self: Module[T], address: string) =
     return
   self.walletAccountService.addressWasShown(address)
 
-method checkIfAddressWasCopied*[T](self: Module[T], value: string) =
-  let walletAcc = self.walletAccountService.getAccountByAddress(value)
-  if walletAcc.isNil:
-    return
-  self.addressWasShown(value)
```

Comment:
Are we sure we no longer need this?

Reply:
It's something I need to discuss with @saledjenic. I'm not sure when exactly it should be called (I presume that for some components holding elided address). But calling it whenever we copy, no matter what, doesn't sound as a good idea neither.

Reply:
All those calls to `addressWasShown` were done to save rpc calls, cause we don't need to check for history transactions if it is a newly added address that we're sure it was not used before. The first time that address is copied or exposed, we're not sure anymore if the user has used it in other wallets.

So I guess we still need this and will need it until we remove displaying history transactions, and that's already ongoing work.